sdk/purview/azure-purview-workflow/azure/purview/workflow/_operations/_operations.py

```diff
@@ -0,0 +1,2734 @@
+                    "lastUpdateTime": "2020-02-20 00:00:00",  # The last update time. Required.
+                    "name": "str",  # The name of a workflow. Required.
+                    "triggers": [
+                        {
+                            "type": "str",  # Required. Known values are:
+                              "when_term_creation_is_requested", "when_term_deletion_is_requested",
+                              "when_term_update_is_requested", "when_terms_import_is_requested",
+                              "when_data_access_grant_is_requested", "when_asset_update_is_requested",
+                              "when_dsar_deletion_is_requested", and "when_dsar_export_is_requested".
+                            "underCollection": "str",  # Optional. The collection name.
+                            "underGlossary": "str",  # Optional. The glossary guid.
+                            "underGlossaryHierarchy": "str"  # Optional. Glossary term
+                              hierarchy path.
+                        }
+                    ],
+                    "updatedBy": "str"  # The person who updated the workflow. Required.
+                }
+        """
+        error_map = {
+            401: ClientAuthenticationError,
+            404: ResourceNotFoundError,
+            409: ResourceExistsError,
+            304: ResourceNotModifiedError,
+        }
+        error_map.update(kwargs.pop("error_map", {}) or {})
+
+        _headers = kwargs.pop("headers", {}) or {}
+        _params = kwargs.pop("params", {}) or {}
+
+        cls: ClsType[JSON] = kwargs.pop("cls", None)
+
+        request = build_purview_workflow_get_workflow_request(
+            workflow_id=workflow_id,
+            api_version=self._config.api_version,
+            headers=_headers,
+            params=_params,
+        )
+        path_format_arguments = {
+            "endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
+        }
+        request.url = self._client.format_url(request.url, **path_format_arguments)
+
+        pipeline_response: PipelineResponse = self._client._pipeline.run(  # pylint: disable=protected-access
+            request, stream=False, **kwargs
+        )
+
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        if response.content:
+            deserialized = response.json()
+        else:
+            deserialized = None
+
+        if cls:
+            return cls(pipeline_response, cast(JSON, deserialized), {})
+
+        return cast(JSON, deserialized)
+
+    @overload
+    def create_or_replace_workflow(
+        self,
+        workflow_id: str,
+        workflow_create_or_update_command: JSON,
```

Comment:
this name will probably get flagged during archboard review

Reply:
May I ask what is flagged? archboard has reviewed this api via api view: https://apiview.dev/Assemblies/Review/43500e48932045aaa9f0b92e18fab1e5